src/scala/scripts/PBGenerateRequest.scala

```diff
@@ -2,48 +2,38 @@ package scripts
 
 import java.nio.file.{Files, Path, Paths}
 
-class PBGenerateRequest(val jarOutput: String, val scalaPBOutput: Path, val scalaPBArgs: List[String], val protoc: Path)
+class PBGenerateRequest(val jarOutput: String, val scalaPBOutput: Path, val scalaPBArgs: List[String], val includedProto: List[(String, String)], val protoc: Path)
 
 object PBGenerateRequest {
 
   def from(args: java.util.List[String]): PBGenerateRequest = {
     val jarOutput = args.get(0)
-    val parsedProtoFiles = args.get(1).split(':').toList.map { rootAndFile =>
-      val parsed = rootAndFile.split(',')
-      val root = parsed(0)
-      val file = if (root.isEmpty) {
-        parsed(1)
-      } else {
-        parsed(1).substring(root.length).stripPrefix("/")
-      }
-      (file, Paths.get(root, file).toString)
-    }
-    // This will map the absolute path of a given proto file
-    // to a relative path that does not contain the repo prefix.
-    // This is to match the expected behavior of
-    // proto_library and java_proto_library where proto files
-    // can import other proto files using only the relative path
-    val imports = parsedProtoFiles.map { case (relPath, absolutePath) =>
-      s"-I$relPath=$absolutePath"
-    }
     val protoFiles = args.get(4).split(':')
+    val includedProto = args.get(1).drop(1).split(':').distinct.map { e =>
+      val p = e.split(',')
+      (p(0), p(1))
```

Comment:
should we just use `Path` here so we don't have to do the conversion later?

Reply:
Yep will update